README.md

```diff
@@ -0,0 +1,85 @@
+# Pig Latin Harness
+
+A programming-language-agnostic test harness for PigLatin translation.
+
+## Usage
+
+```bash
+./harness.sh <cmd>
+```
+
+Where `<cmd>` is a command line utility that reads from the incomping pipeline and writes to standard out.
+
+For example, if you had the input file `english.txt`
+
+```
+This is a
+working Pig Latin
+translator!
+```
+
+Then you would use the translator utility `translate.py` like
+
+```bash
+$ translate.py < english.txt
+# => Isthay isway away
+# => orkingway Igpay Atinlay
+# => anslatortray!
+```
+
+### Example
+
+```ruby
+# pig_latin.rb
+#!/usr/bin/env ruby
+
+module PigLatin
+  def self.translate(string)
+    string.gsub(/\w+/) do |word|
+      word.sub(/^(y|[^aeioquy]*(?:qu)?)(.*)/i) do
+        "#{$2}#{$1}#{$1 == '' ? 'w' : ''}ay"
+      end.tap { |w| w.capitalize! if word =~ /^[A-Z]/ }
+    end
+  end
+end
```

Comment:
My assumption was that this repo was gonna be used by a test proctor, not a test taker. And giving the proctor the tools to use this harness from end-to-end would allow them suss out any hiccups before administering the test to someone else.